crypto/salsa20/xmm6int/salsa20_xmm6int-sse2.c

```diff
@@ -1,4 +1,4 @@
-#if defined(ANDROID) || defined(RPI)
+#if defined(ANDROID) || defined(iOS) || defined(RPI)
```

Comment:
This should read
```c
#if __SSE2__
```
instead, using inbuild compiler support to detect whether the SSE2 instruction is present, rather then relying on explicit build flags/defines.